spec/runtime/setup_spec.rb

```diff
@@ -108,6 +108,38 @@
     end
   end
 
+  context "load order" do
+    after do
+      ENV.delete 'RUBYLIB'
```

Comment:
double quotes, and this shouldn't be necessary because of https://github.com/bundler/bundler/blob/0647f9645add95a9ec522b0f8a880b1745d67112/spec/spec_helper.rb#L117